src/view/QuestionAttemptTemplate.java

```diff
@@ -0,0 +1,282 @@
+import java.awt.BorderLayout;
+import java.awt.Dimension;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.awt.GridLayout;
+import java.awt.Insets;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import javax.swing.BorderFactory;
+import javax.swing.ButtonGroup;
+import javax.swing.JButton;
+import javax.swing.JDialog;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JRadioButton;
+import javax.swing.border.EmptyBorder;
+
+/**
+ * Form to show a question and navaigation buttion and to give user
+ * functionality to select one of the options.
+ *
+ * @author Sarthak Tiwari
+ * @version 1.0
+ */
+public final class QuestionAttemptTemplate extends JDialog {
+
+    private static QuestionAttemptTemplate instance = null;
+
+    private static JLabel questionDescription;
+
+    private static JRadioButton option1;
+    private static JRadioButton option2;
+    private static JRadioButton option3;
+    private static JRadioButton option4;
+    private static ButtonGroup options;
+
+    private static JButton next;
+    private static JButton giveup;
+
+    private static short correctAnswer = 0;
+    private static int isCorrectAnswer;
```

Comment:
variable name starting with 'is' like in this case, 'isCorrectAnswer' should be boolean not int.

Reply:
Umm.. it still represents whether the answer is correct or not.. can you suggest some name ?

Reply:
Does 'reponseStatus' sound okay ?

Reply:
yeah ok. Sounds fine.